templates/react-native/src/permission.ts.twig

```diff
@@ -0,0 +1,22 @@
+export class Permission {
+
+    static read = (role: string): string => {
+        return `read("${role}")`
+    }
+
+    static write = (role: string): string => {
+        return `write("${role}")`
+    }
```

Comment:
I don't think `write` is supported anymore

Reply:
I think it should be, it's in all the SDKs